### PR TITLE
Added clone_from_snapshot_builder.go

### PR DESCRIPTION
**Pull request by dsu-igeek**

### CODE REVIEW

**Review by deepakkinni — APPROVED** [review]
lgtm

File: pkg/builder/clone_from_snapshot_builder.go [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+type CloneFromSnapshotBuilder struct {
+	object *backupdriverv1.CloneFromSnapshot
+}
+
+func ForCloneFromSnapshot(ns, name string, labels map[string]string) *CloneFromSnapshotBuilder {
+	return &CloneFromSnapshotBuilder{
+		object: &backupdriverv1.CloneFromSnapshot{
+			TypeMeta: metav1.TypeMeta{
+				APIVersion: backupdriverv1.SchemeGroupVersion.String(),
+				Kind:       "Snapshot",
```
Comment (deepakkinni) [inline pkg/builder/clone_from_snapshot_builder.go:34]: change Kind type

Comment (dsu-igeek) [reply pkg/builder/clone_from_snapshot_builder.go]: Done, thanks

**Comment by svcbot-qecnsdp** [comment]
Sorry, some of the pre-check tests are failing for this PR ! 

Please check the Jenkins job for more details
https://container-dp.svc.eng.vmware.com/job/CNSDP-CI/230/

**Comment by svcbot-qecnsdp** [comment]
Sorry, some of the pre-check tests are failing for this PR ! 

Please check the Jenkins job for more details
https://container-dp.svc.eng.vmware.com/job/CNSDP-CI/231/

**Comment by svcbot-qecnsdp** [comment]
Congratulations !! 

All the pre-check tests passed
https://container-dp.svc.eng.vmware.com/job/CNSDP-CI/232/

**Pull request merged**